lumicks/pylake/kymo.py

```diff
@@ -413,37 +414,60 @@ def plot(
 
         return image_handle
 
-    def plot_with_force(
+    def plot_with_channels(
         self,
-        force_channel,
-        color_channel,
+        channels,
+        color_channel="rgb",
+        *,
         aspect_ratio=0.25,
-        reduce=np.mean,
         kymo_args=None,
         adjustment=no_adjustment,
+        title_vertical=False,
+        labels=None,
```

Comment:
I have no idea how hard this is, but is it possible to allow for other kinds of plot arguments, such as the color of the plot line? Not essential, but would be nice. The most common things users may want to change the color of the line and maybe to add a legend.

Reply:
Any remaining keyworded arguments get forwarded to `Slice.plot()`.

So you could already do:
```
kymo.plot_with_channels(
    [
        h5_file.force1x.downsampled_by(100),
        h5_file["Photon count"]["Green"].downsampled_over(kymo.line_timestamp_ranges(), reduce=np.sum),
    ],
    "rgb",
    adjustment=lk.ColorAdjustment(5, 98, "percentile"),
    aspect_ratio=0.2,
    title_vertical=False,
    color="blue",
)
```

resulting in

<img width="579" alt="image" src="https://github.com/lumicks/pylake/assets/19836026/df3b00d6-d177-4493-8d4a-7b28cfbc14af">

for example.

What do you mean by a legend? Every graph has its own subplot so I don't see where one would add a legend. Or do you mean a scalebar?

That is possible since you can also pass kymograph arguments. Unfortunately, that is a little more complicated (because either the kymo customization or the line plots can be forwarded easily, but not both).

```
kymo.plot_with_channels(
    [
        h5_file.force1x.downsampled_by(100),
        h5_file["Photon count"]["Green"].downsampled_over(kymo.line_timestamp_ranges(), reduce=np.sum),
    ],
    "rgb",
    adjustment=lk.ColorAdjustment(5, 98, "percentile"),
    aspect_ratio=0.2,
    title_vertical=False,
    kymo_args={"scale_bar": lk.ScaleBar(10.0, 5.0)},
    color="blue",
)
```

returns:

<img width="575" alt="image" src="https://github.com/lumicks/pylake/assets/19836026/e053a058-cf76-4266-ae6a-60742e873c89">

Reply:
I actually meant individual colors. (eg, plot the green photon counts in green, the force in black). And I meant a legend, but I agree that that does not add much value.

Reply:
Maybe I should add it as example in the docs.

Reply:
Alright. I've added a `colors` and `scale_bar` keyword arg to allow for easier customization. I've also added a section that demonstrates these to the docs.

Reply:
Alright, `colors` and `labels` are now exposed as well.
<img width="842" alt="image" src="https://github.com/lumicks/pylake/assets/19836026/5300f39a-1bf6-40b8-b739-d5bb96acd463">